### ISSUE TITLE
Element mutation polyfills should work on all types of parentNode

### ISSUE THREAD

**Issue by triblondon** [polyfillpolyfill/polyfill-service#281]
@jonathantneal in #254:

I would like to address another issue, too. These polyfills should target childnode and parentnode elements, which in HTML roughly translate to:

`ParentNode` => `Document`, `DocumentFragment` (`HTMLDocument` in IE8), `Element`

`ChildNode` => `Element`, `Text`, `HTMLCommentElement`, `DocumentType` (`HTMLCommentElement` in IE8)

`ParentNode` is a kind of low-level object that does not actually exist in the browser. It represents Document, DocumentFragment and Element simultaneously.

```
ParentNode.prototype.prepend = function prepend() {};
```

Except I see no sane way to compile that into what is required:

```
Document.prototype.prepend = DocumentFragment.prototype.prepend = Element.prototype.prepend = function prepend() {};
```

**Comment by mislav** [comment]
While fixing this, it could be a good idea to handle #286 in the same go. Thanks

**Comment by triblondon** [comment]
Resolved by #288
